From cbb16d5e2c856178d27e1c133d9b5b095603024c Mon Sep 17 00:00:00 2001
From: Domenico Andreoli <cavok@debian.org>
Date: Mon, 20 Jun 2005 11:34:16 +0200
Subject: changes in pycurl_7.14.0-1.diff.gz

---
 src/pycurl.c | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/src/pycurl.c b/src/pycurl.c
index 2a59c0d..36aacd9 100644
--- a/src/pycurl.c
+++ b/src/pycurl.c
@@ -1178,6 +1178,11 @@ do_curl_setopt(CurlObject *self, PyObject *args)
         return Py_None;
     }
 
+#ifndef PY_LONG_LONG
+#define PY_LONG_LONG long long
+#define PY_LONG_LONG_CAVOK_HACK
+#endif
+
     /* Handle the case of long arguments (used by *_LARGE options) */
     if (PyLong_Check(obj)) {
         PY_LONG_LONG d = PyLong_AsLongLong(obj);
@@ -1199,6 +1204,11 @@ do_curl_setopt(CurlObject *self, PyObject *args)
         return Py_None;
     }
 
+#ifdef PY_LONG_LONG_CAVOK_HACK
+#undef PY_LONG_LONG
+#undef PY_LONG_LONG_CAVOK_HACK
+#endif
+
 #undef IS_LONG_OPTION
 #undef IS_OFF_T_OPTION
 
