From a4242dd2babfabd9eef772424d35ddb257dacbf0 Mon Sep 17 00:00:00 2001
From: Domenico Andreoli <cavok@debian.org>
Date: Tue, 8 Aug 2006 08:22:16 +0200
Subject: changes in pycurl_7.15.4.2-2.diff.gz

---
 src/pycurl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/pycurl.c b/src/pycurl.c
index 8f48d10..31d0b1a 100644
--- a/src/pycurl.c
+++ b/src/pycurl.c
@@ -416,7 +416,7 @@ static int pycurl_ssl_mutex_destroy(void **m)
 
 static int pycurl_ssl_mutex_lock(void **m)
 {
-    return PyThread_acquire_lock(*((PyThread_type_lock *) m), 1);
+    return !PyThread_acquire_lock(*((PyThread_type_lock *) m), 1);
 }
 
 static int pycurl_ssl_mutex_unlock(void **m)
