From d9c5a1a3a324a11dbbda6fa0132e1f7228a783f3 Mon Sep 17 00:00:00 2001
From: Domenico Andreoli <cavok@debian.org>
Date: Mon, 14 May 2007 01:17:33 +0200
Subject: changes in pycurl_7.16.2.1-2.diff.gz

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index be2c41c..9093a4a 100644
--- a/setup.py
+++ b/setup.py
@@ -17,7 +17,7 @@ from distutils.util import split_quoted
 from distutils.version import LooseVersion
 
 include_dirs = []
-define_macros = []
+define_macros = [ ('HAVE_CURL_GNUTLS', 1) ]
 library_dirs = []
 libraries = []
 runtime_library_dirs = []
