From df561a58bf6d84c8796e9f6fa317bc482490cd5e Mon Sep 17 00:00:00 2001
From: Sandro Tosi <matrixhasu@gmail.com>
Date: Mon, 2 Jun 2008 19:17:08 +0200
Subject: =?UTF-8?q?setup.py=0APatch-Name:=2010=5Fsetup.py?=

---
 setup.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index d0df478..08b0cd1 100644
--- a/setup.py
+++ b/setup.py
@@ -17,7 +17,7 @@ from distutils.util import split_quoted
 from distutils.version import LooseVersion
 
 include_dirs = []
-define_macros = []
+define_macros = [ ('HAVE_CURL_GNUTLS', 1) ]
 library_dirs = []
 libraries = []
 runtime_library_dirs = []
@@ -188,7 +188,6 @@ setup_args = get_kw(
     maintainer_email="kjetilja at gmail.com, markus at oberhumer.com",
     url="http://pycurl.sourceforge.net/",
     license="LGPL/MIT",
-    data_files=get_data_files(),
     ext_modules=[ext],
     long_description="""
 This module provides Python bindings for the cURL library.""",
