From 5b70601b4693421b3cdd5d313791646c99bc9dd6 Mon Sep 17 00:00:00 2001
From: Domenico Andreoli <cavok@debian.org>
Date: Mon, 7 Mar 2005 14:51:16 +0100
Subject: changes in pycurl_7.13.1-1.diff.gz

---
 src/pycurl.c | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/src/pycurl.c b/src/pycurl.c
index 6f5f699..ce1749c 100644
--- a/src/pycurl.c
+++ b/src/pycurl.c
@@ -1175,6 +1175,11 @@ do_curl_setopt(CurlObject *self, PyObject *args)
         return Py_None;
     }
 
+#ifndef PY_LONG_LONG
+#define PY_LONG_LONG long long
+#define PY_LONG_LONG_CAVOK_HACK
+#endif
+
     /* Handle the case of long arguments (used by *_LARGE options) */
     if (PyLong_Check(obj)) {
         PY_LONG_LONG d = PyLong_AsLongLong(obj);
@@ -1196,6 +1201,11 @@ do_curl_setopt(CurlObject *self, PyObject *args)
         return Py_None;
     }
 
+#ifdef PY_LONG_LONG_CAVOK_HACK
+#undef PY_LONG_LONG
+#undef PY_LONG_LONG_CAVOK_HACK
+#endif
+
 #undef IS_LONG_OPTION
 #undef IS_OFF_T_OPTION
 
