From 7e232eecf5078c81d910a03717b440a5e2af7416 Mon Sep 17 00:00:00 2001
From: Barry Warsaw <barry@python.org>
Date: Tue, 14 Oct 2014 16:25:57 -0400
Subject: upstream issue 210

---
 src/easy.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/easy.c b/src/easy.c
index e521de4..2a7d361 100644
--- a/src/easy.c
+++ b/src/easy.c
@@ -929,7 +929,11 @@ debug_callback(CURL *curlobj, curl_infotype type,
     }
 
     /* run callback */
+#if PY_MAJOR_VERSION >= 3
+    arglist = Py_BuildValue("(iy#)", (int)type, buffer, (int)total_size);
+#else
     arglist = Py_BuildValue("(is#)", (int)type, buffer, (int)total_size);
+#endif
     if (arglist == NULL)
         goto verbose_error;
     result = PyEval_CallObject(self->debug_cb, arglist);
