Description: Use "find -perm /x" instead of "find -perm +x".
Author: Andreas Metzler <ametzler@debian.org>
Bug-Debian: https://bugs.debian.org/808922
Origin: vendor

--- pycurl-7.19.5.3.orig/Makefile
+++ pycurl-7.19.5.3/Makefile
@@ -175,8 +175,8 @@ rsync-check:
 # NOTE: rsync targets depend on www.
 rsync-prepare:
 	chgrp $(CHMOD_VERBOSE) -R pycurl $(BUILD_WWW)
-	chmod $(CHMOD_VERBOSE) g+r `find $(BUILD_WWW) -perm +400 -print`
-	chmod $(CHMOD_VERBOSE) g+w `find $(BUILD_WWW) -perm +200 -print`
+	chmod $(CHMOD_VERBOSE) g+r `find $(BUILD_WWW) -perm /400 -print`
+	chmod $(CHMOD_VERBOSE) g+w `find $(BUILD_WWW) -perm /200 -print`
 	chmod $(CHMOD_VERBOSE) g+s `find $(BUILD_WWW) -type d -print`
 ##	chmod $(CHMOD_VERBOSE) g+rws `find $(BUILD_WWW) -type d -perm -770 -print`
 	chmod $(CHMOD_VERBOSE) g+rws `find $(BUILD_WWW) -type d -print`
