--- a/Makefile
+++ b/Makefile
@@ -145,9 +145,9 @@ run-quickstart:
 docs: build
 	mkdir -p build/docstrings
 	for file in doc/docstrings/*.rst; do tail -n +3 $$file >build/docstrings/`basename $$file`; done
-	PYTHONSUFFIX=$$(python -V 2>&1 |awk '{print $$2}' |awk -F. '{print $$1 "." $$2}') && \
+	PYTHONSUFFIX=$$(python3 -V 2>&1 |awk '{print $$2}' |awk -F. '{print $$1 "." $$2}') && \
 	PYTHONPATH=$$(ls -d build/lib.*$$PYTHONSUFFIX):$$PYTHONPATH \
-	sphinx-build doc build/doc
+	python3 -m sphinx doc build/doc
 	rst2html README.rst build/doc/README.html
 
 # Rebuild all documentation.
